### PR TITLE
Small enhancement on function execution time statistic

### PR DESCRIPTION
**Pull request by lum1n0us**
- support cpu time and real time. 
- support children execution time.

TODOs:
- [x] ~need~ customer confirm~ation abou~t cpu time ~or real time~
- [x] fix aot name section emission failure
- [x] multi-threads

### CODE REVIEW

File: core/shared/utils/bh_atomic.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #define _BH_ATOMIC_H
 
 #include "gnuc.h"
+#include "platform_common.h"
```
Comment (wenyongh) [inline core/shared/utils/bh_atomic.h:10]: Seems no need to include this file?

File: core/shared/utils/bh_atomic.h [hunk 2]
```diff
@@ -101,6 +112,62 @@ typedef uint16 bh_atomic_16_t;
 #endif
 #endif
 
+#if BH_ATOMIC_64_IS_ATOMIC != 0
+
+#define BH_ATOMIC_64_LOAD(v) __atomic_load_n(&(v), __ATOMIC_SEQ_CST)
```
Comment (wenyongh) [inline core/shared/utils/bh_atomic.h:117]: BH_ATOMIC_64_XXX macros are not used in this PR, not sure whether we should change this file? Had better add them when needed?

File: core/iwasm/interpreter/wasm_runtime.c [hunk 3]
```diff
@@ -2398,53 +2398,104 @@ wasm_call_function(WASMExecEnv *exec_env, WASMFunctionInstance *function,
...
+
+double
+wasm_get_wasm_func_exec_time(const WASMModuleInstance *inst,
+                             const char *func_name)
+{
+    unsigned i;
+    for (i = 0; i < inst->e->function_count; i++) {
+        char *name_in_wasm = get_func_name_from_index(inst, i);
+        if (name_in_wasm
+            && strncmp(name_in_wasm, func_name, strlen(func_name)) == 0) {
```
Comment (wenyongh) [inline core/iwasm/interpreter/wasm_runtime.c:2490]: Should use `strcmp(name_in_wasm, func_name) == 0`?

**Review by wenyongh — COMMENTED** [review]
LGTM

**Review by no1wudi — APPROVED** [review]

**Pull request merged**